gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

Comment:
yes, this change is not necessary

Reply:
I forgot to change it back to 8.8. I used Gradle 8.6 because the new version failed to build with IntelliJ Idea 2024 and I don't know why it happend. Anyway, I'll change it back to 8.6